TagsEditText/src/main/java/mabbas007/tagsedittext/TagsEditText.java

```diff
@@ -274,9 +274,10 @@ public void onRestoreInstanceState(Parcelable state) {
             mIsSpacesAllowedInTags = bundle.getBoolean(ALLOW_SPACES_IN_TAGS, mIsSpacesAllowedInTags);
 
             mLastString = bundle.getString(LAST_STRING);
-            Tag[] tags = (Tag[]) bundle.getParcelableArray(TAGS);
```

Comment:
We can't cast arrays like that, please see related stackoverflow question:
http://stackoverflow.com/questions/8745893/i-dont-get-why-this-classcastexception-occurs